app/views/practices/form/introduction.html.erb

```diff
@@ -63,6 +63,20 @@
               </p>
             </div>
 
+          <div class="practice-editor-rect-svg border-top-05 border-primary-dark"></div>
+            <div class="margin-bottom-5 margin-top-3">
+              <div>
+                <%= f.label :short_name, 'Acronym ', class: 'usa-label text-bold display-block margin-top-0 margin-bottom-2' %>
+                <span>Type the acronym for your practice.</span><!--&nbsp;<span id="practice-editor-tagline-character-counter" class="text-base-light">(0/150 characters)</span>-->
+              </div>
+              <div>
+                <%= f.text_field :short_name, class: "grid-col-3 usa-input #{ @practice.errors[:short_name].any? ? 'usa-input--error' : '' } display-block practice-editor-short_name-input", required: false %>
+              </div>
+              <p class="usa-error-message <%= @practice.errors[:short_name].any? ? 'fas fa-exclamation-circle fon' : 'display-none' %>">
```

Comment:
I know you didn't add this but I just noticed a ` fon` class... Is this actually a class haha just making sure since I've never seen that before 😅

Reply:
I don't think so. I'll go ahead and remove it hehe